examples/next/docs/react/demo/src/renderers/ProgressBar.tsx

```diff
@@ -1,10 +1,21 @@
-import React from "react";
-import { addClassWhenNeeded, RendererProps } from "./utils";
+import React from 'react';
+import { getRangeValue, HandsontableProps } from './utils';
+import { MESSAGE } from '../constants';
 
-export function ProgressBarRenderer(props: RendererProps) {
-  addClassWhenNeeded(props);
+const minAllowedValue = 0;
+const maxAllowedValue = 100;
 
+export const ProgressBarRenderer = (props: HandsontableProps) => {
+  if (props.cellProperties.valid !== false) {
```

Comment:
@maciejbo-handsontable 
I found one additional issue with the demo - when you pass invalid data for the progress bar or stars columns, they're not marked as invalid. 
That happens because Handsontable doesn't trigger validation by default when loading data.
To get around it it, you can manually trigger the validator if it wasn't triggered already.

Take a look at my suggestion, and if you think it works all right, could you implement it for the stars renderer as well?

```suggestion
export const ProgressBarRenderer = (props: HandsontableProps) => {
  let isValid = props.cellProperties.valid;

  // Run the validator for the cell at initization.
  if (isValid === void 0) {
    (props.cellProperties.validator as Function)(props.value, (isValueValid: boolean) => {
      isValid = isValueValid;
    });
  }

  if (isValid === true) {
```